modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/AppContext.es.js

```diff
@@ -485,20 +502,11 @@ const createReducer = (dataLayoutBuilder) => {
 				};
 			}
 			case UPDATE_EDITING_LANGUAGE_ID: {
-				const {dataDefinition} = state;
+				const editingLanguageId = action.payload;
```

Comment:
This action.payload could be used directly at editingLanguageId ??

Just to keep the same code from line 423

Reply:
You're right, Keven. I'll change the code.